README.md

```diff
@@ -34,18 +34,14 @@ test('should result to the answer', t => {
 ```
 
 You can run your test with
-1. Node: ``node ./myTestFile`
+1. Node: ``node --experimental-modules --es-module-specifier-resolution=node ./myTestFile.js`
 2. In the browser ``<script type="module" src="./myTestFile.js></script>`` identically
 
 Moreover zora does not use specific platform API which should make it transparent to most of your tools such module bundlers or transpilers.
 
 In few words:
 > Zora is Ecmascript, no less, no more.
 
-### Node config
-
-In Node 12 you should add flags `--experimental-modules --es-module-specifier-resolution=node` and add `"type": "module"` to your project.json. In smaller versions use full path to zora library and .mjs extension in all your files.
-
```

Comment:
or bundle your test program as commonjs

Reply:
cjs is not ES6, it’s specific markup for node. As I understand your library is unique, because it work without any bundler.

Reply:
Yes. However in a usage note (by the way your paragraph should probably be moved to the usage section) you need to be pragmatic and think on how your users actually use the library. Assuming everybody uses node 12 is probably wrong. For example, in serverless environment the main providers still use node 10 if I am not wrong.

I see more something like:
usage
* Node
  * with ecmascript module syntax
    * Node < 12: bundler to cjs
    * Node 12: expermental flags
  * with cjs syntax: just run with node
* Browsers
  * es6 module
  * legacy browsers: bundler to iife

Something like that

Reply:
I am added your test runner here https://testanything.org/producers.html .
Please create an article on medium/dev.to

Reply:
@munrocket 
[first article](https://dev.to/lorenzofox3/tools-and-the-design-of-a-testing-experience-2mdc) of an ongoing serie ! Thanks for the idea

Reply:
@lorenzofox3 example how to use zora with puppeteer and ES6 #47